test/pkg/gitea/test.go

```diff
@@ -369,7 +369,7 @@ func NewPR(t *testing.T, topts *TestOpts) func() {
 		}
 		assert.Assert(t, len(events.Items) != 0, "events expected in case of failure but got 0")
 	} else if !topts.SkipEventsCheck {
-		assert.Assert(t, len(events.Items) == 0, fmt.Sprintf("no events expected but got %v in %v ns, items: %+v", len(events.Items), topts.TargetNS, events.Items))
+		assert.Assert(t, len(events.Items) == 1, fmt.Sprintf("no events expected but got %v in %v ns, items: %+v", len(events.Items), topts.TargetNS, events.Items))
```

Comment:
i don't think that's right... there is something happening with regard to cancellation and we can't just assume == 1 here...